src/main/drivers/barometer/barometer_bmp085.c

```diff
@@ -119,11 +120,11 @@ static bool bmp085InitDone = false;
 STATIC_UNIT_TESTED uint16_t bmp085_ut;  // static result of temperature measurement
 STATIC_UNIT_TESTED uint32_t bmp085_up;  // static result of pressure measurement
 
-static void bmp085_get_cal_param(void);
-static void bmp085_start_ut(void);
-static void bmp085_get_ut(void);
-static void bmp085_start_up(void);
-static void bmp085_get_up(void);
+static void bmp085_get_cal_param(busDevice_t *pBusdev);
```

Comment:
Betaflight style is not to use `p` for pointers, so `bmp085_get_cal_param(busDevice_t *busDev);`